tests/components/test_tag.py

```diff
@@ -52,7 +52,7 @@ def test_is_valid_prop(prop: Var, valid: bool):
 
 def test_add_props():
     """Test that the props are added."""
-    tag = Tag().add_props(key="value", key2=42, invalid=None, invalid2={})
+    tag: str = Tag().add_props(key="value", key2=42, invalid=None, invalid2={})
```

Comment:
```suggestion
    tag= Tag().add_props(key="value", key2=42, invalid=None, invalid2={})
```

similarly, revert the other similar changes in this file